app/controllers/v1/people_controller.rb

```diff
@@ -1,18 +1,14 @@
 class V1::PeopleController < V1::BaseController
 
   def create
-    person = Integration::NationBuilder.create_or_update_person(attributes: params[:person])
-    render json: {id: person['id']}, status: 200
+    person = Person.create_or_update(person_params)
+    render json: { id: person.id }, status: 200
```

Comment:
returning `person.id` is now meaningless, but I didn't know what to replace it with so I just left it for now